Arrow/Sources/Arrow/ArrowType.swift

```diff
@@ -366,6 +417,24 @@ public class ArrowType {
             return ArrowTypeTime64(.microseconds)
         } else if  from == "ttn" {
             return ArrowTypeTime64(.nanoseconds)
+        } else if from.starts(with: "ts") {
+            let components = from.split(separator: ":", maxSplits: 1)
+            guard let unitPart = components.first, unitPart.count == 3 else {
+                throw ArrowError.notImplemented
```

Comment:
It seems that this is an invalid case like `tsXX`. Can we use more suitable exception than `ArrowError.notImplemented`?